main/views.py

```diff
@@ -23,6 +23,8 @@ def _index(request):
         "release_version": settings.VERSION,
         "sentry_dsn": settings.SENTRY_DSN,
         "gdrive_enabled": is_gdrive_enabled(),
+        "features": settings.FEATURES,
```

Comment:
ocw-studio only has one feature flag at this time, namely `FEATURE_USE_LOCAL_STARTERS`. So, any kind of filter here right now is unnecessary IMO.